### PR TITLE
[Task] Update our top level security policy

### PR DESCRIPTION
**Pull request by unlikelyzero**
Closes: https://github.com/nasa/openmct/issues/4424

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [x] Is this change backwards compatible? For example, developers won't need to change how they are calling the API or how they've extended core plugins such as Tables or Plots.

### Author Checklist

* [x] Changes address original issue?
* [ ] Unit tests included and/or updated with changes?
* [ ] Command line build passes?
* [ ] Has this been smoke tested?
* [ ] Testing instructions included in associated issue?

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -1,23 +0,0 @@
-<!--- This is for filing enhancements or features. If you have a general -->
```
Comment (akhenry) [inline .github/ISSUE_TEMPLATE/feature_request.md:1]: This meant to be deleted?

**Review by akhenry — CHANGES_REQUESTED** [review]
I might just be missing something, but it looks to me as though the feature request template has been removed, and a new "maintenance" template has been added, and it seems like we need both?

**Comment by unlikelyzero** [comment]
@akhenry sorry, i wasn't clear in the description. We have an old unused template (from our original attempt at implementing templates. Because it wasn't configured correctly, it's just unused.

This PR adds a new maintenance type in addition to remove the improperly formatted one.

**Comment by akhenry** [comment]
### Reviewer Checklist

1. Changes appear to address issue? Y
2. Appropriate unit tests included? N/A
3. Code style and in-line documentation are appropriate? Y
4. Commit messages meet standards? Y
5. Has associated issue been labelled `unverified`? (only applicable if this PR closes the issue) Y

**Review by akhenry — APPROVED** [review]

**Pull request merged**